src/main/java/at/ac/tuwien/kr/alpha/config/CommandLineParser.java

```diff
@@ -107,6 +112,17 @@
 			.desc("disable the deletion of (learned, little active) nogoods (default: "
 					+ SystemConfig.DEFAULT_DISABLE_NOGOOD_DELETION + ")")
 			.build();
+	private static final Option OPT_GROUNDER_TOLERANCE_CONSTRAINTS = Option.builder("gtc").longOpt("grounderToleranceConstraints")
+			.desc("grounder tolerance for constraints (default: " + SystemConfig.DEFAULT_GROUNDER_TOLERANCE_CONSTRAINTS + ")")
+			.hasArg().argName("tolerance")
+			.build();
+	private static final Option OPT_GROUNDER_TOLERANCE_RULES = Option.builder("gtr").longOpt("grounderToleranceRules")
+			.desc("grounder tolerance for rules (default: " + SystemConfig.DEFAULT_GROUNDER_TOLERANCE_RULES + ")")
+			.hasArg().argName("tolerance")
+			.build();
+	private static final Option OPT_GROUNDER_NO_INSTANCE_REMOVAL = Option.builder("acc").longOpt("enableAccumulator")
```

Comment:
The renaming to accumulator and permissive makes the code and parameters much nicer, great!
Could you also rename the option to `OPT_GROUNDER_ACCUMULATOR_ENABLED`?

Reply:
Sure thing, I just missed this occurrence before.